recipes-connectivity/prplmesh/prplmesh/002-allow_redefine_beerocks_vars_from_cmdline.patch

```diff
@@ -0,0 +1,166 @@
+commit 9f1aa2d7bfcc74a38ebab39892283468b0c7472d
```

Comment:
This one was merged upstream.

Reply:
Used prplMesh commit will be updated in separate task https://jira.prplfoundation.org/browse/PPM-281